marketplace/deployer_util/provision.py

```diff
@@ -80,6 +83,8 @@ def process(schema, values, deployer_image, deployer_entrypoint):
           schema, prop, app_name=app_name, namespace=namespace)
       props[prop.name] = value
       manifests += sc_manifests
+    elif prop.xtype == 'APPLICATION_UID':
```

Comment:
This is not the right place for it. We want to do this at configuration expansion time, similar to [this](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/894cffecaa1cf9cb76cf7d5fb89a9dc52a3437f6/marketplace/deployer_util/expand_config.py#L69). The expansion is executed [here](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/183d98a20cb2d49ca23d2181c665a189b5d03969/marketplace/deployer_util/deploy.sh#L40).

That way we shouldn't need to change anything in the backend.

Reply:
Discussed approach offline.

Done!